src/videouplinkbandwidthpolicy/VideoUplinkBandwidthPolicy.ts

```diff
@@ -62,11 +62,11 @@ export default interface VideoUplinkBandwidthPolicy {
   /**
    * Set a reference to the current transceiver controller.
    * This should be called whenever a transceiver controller is available when the call starts.
-   * The default audio video controllers calls this at the end of connection.
+   * The default audio video controllers calls this when the connection starts and clear it when the connection ends.
```

Comment:
"… calls this with a `TransceiverController` when the meeting starts, and calls it with `undefined` when the meeting ends. If a meeting is stopped and started repeatedly, this pair of calls will be repeated to match. All calls to `updateTransceiverController` will occur between this pair of calls."